src/cli/commands/lint.ts

```diff
@@ -14,13 +15,25 @@ const formatOptions = Object.values(OutputFormat);
 
 const lintCommand: CommandModule = {
   describe: 'lint JSON/YAML documents from files or URLs',
-  command: 'lint <documents..>',
+  command: 'lint [documents..]',
   builder: yargs =>
     yargs
+      .strict()
       .positional('documents', {
         description:
           'Location of JSON/YAML documents. Can be either a file, a glob or fetchable resource(s) on the web.',
         type: 'string',
+        coerce(values) {
+          if (values.length > 0) {
+            return values;
+          }
+
+          if (process.env.isTTY || process.stdin.isTTY) {
```

Comment:
Im not sure I understand what is happening here? What is TTY and why does it need to be true?

Reply:
I added comments explaining that. LMK if it helps.